### PR TITLE
fix: bring back callback_manager to AgentRunner stream_chat

### PR DESCRIPTION
**Pull request by nerdai**
# Description

- Could be a reason that some traces have disconnected since v0.10.20
- Adding back callback manager to stream_chat which should've been the case to begin with (my bad)
- NOTE: this doesn't fix the issue with LlamaDebugHandler not working (still working on this)

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] I stared at the code and made sure it makes sense

### CODE REVIEW

File: llama-index-core/llama_index/core/agent/runner/base.py [hunk 1]
```diff
@@ -628,6 +628,7 @@ async def achat(
         return chat_response
 
     @dispatcher.span
+    @trace_method("chat")
```
Comment (logan-markewich) [inline llama-index-core/llama_index/core/agent/runner/base.py:631]: I noticed this is the only function with a dispatcher span? 👀

File: llama-index-core/storage/default__vector_store.json
Comment (logan-markewich) [inline llama-index-core/storage/default__vector_store.json:1]: I think this folder was committed by accident (some test is creating this, never tracked it down lol)

Comment (nerdai) [reply llama-index-core/storage/default__vector_store.json:1]: aw darn! lol one sec. thanks!

File: llama-index-core/llama_index/core/agent/runner/base.py (hunk 1 above)
Comment (nerdai) [reply llama-index-core/llama_index/core/agent/runner/base.py:631]: This was just for initial testing with stream_chat stuff. We need to expand coverage here, but not trying to change much yet until the backwards_compat stuff has been resolved. Hope that makes sense.

Comment (nerdai) [reply llama-index-core/llama_index/core/agent/runner/base.py:631]: We could just remove it until we handle streaming and agent runner completely with events and spans.

Comment (logan-markewich) [reply llama-index-core/llama_index/core/agent/runner/base.py:631]: thats fair, we can leave it for now

**Review by logan-markewich — APPROVED** [review]

**Pull request merged**